src/ansys/fluent/core/__init__.py

```diff
@@ -12,14 +12,46 @@
 logging.configure_env_var()
 
 from ansys.fluent.core._version import __version__  # noqa: F401
+from ansys.fluent.core.file_session import (  # noqa: F401
+    InvalidFieldName,
+    InvalidFieldNamePrefix,
+    SurfaceNamesIDsNotProvided,
+)
```

Comment:
These exceptions are only raised and used inside `file_session.py` module, why expose them here?

My initial thought is that none of these exceptions should be exposed globally like this in `__init__`, as they are only used in individual modules

I would suggest moving the exception class definitions to the top of their respective modules (rather than having them at the bottom), so that you don't need to import them

Reply:
Yes, this is also better way. I just followed existing behavior from `pyansys-units` repo. It will also resolve import issues. Thank you.